doc/release-notes/0.35/0.35.md

```diff
@@ -49,45 +49,38 @@ The following table covers notable changes in v0.35.0. Further information about
 <tbody>
 
 <tr>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
-<td valign="top"></td>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/15555">#15555</a></td>
+<td valign="top"> Java&trade; dump output contains a new `Initializing thread: <thread_name>` string in the existing `3XMTHREADBLOCK` line.</td>
```

Comment:
The back quotes aren't doing anything in the .md file. In other places `<tt>` is used instead. All the places that use back quotes need to be updated.

Reply:
Corrected.